cernopendata/modules/search_ui/helpers.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Open Data Portal.
+# Copyright (C) 2021 CERN.
+#
+# CERN Open Data Portal is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Open Data Portal is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""CERN Open Data Invenio-Search-UI helpers."""
+
+from flask import current_app
+from invenio_search_ui.views import SearchAppInvenioRestConfigHelper
+
+
+class CODSearchAppInvenioRestConfigHelper(SearchAppInvenioRestConfigHelper):
+    """Configuration generator for Invenio-Search-JS.
+
+    Using the existing configuration from Invenio-Records-REST we can
+    prefill most of the needed information required for the initialisation
+    of SearchApp.
+    """
+
+    @property
+    def aggs(self):
+        """Format the aggs configuration to be used in React-SearchKit JS.
+
+        Overridden to support multiple ES aggregation types.
+        :returns: A list of dicts for React-SearchKit JS.
+        """
+        search_index = self._rest_config["search_index"]
+        aggs = (
+            current_app.config.get("RECORDS_REST_FACETS", {})
+            .get(search_index, {})
+            .get("aggs", {})
+        )
+        aggs_list = []
+        for k, v in aggs.items():
+            agg_type = [k for k in v.keys() if k != "aggs"][0]
```

Comment:
Minor: I would perhaps give more descriptive names to the variables like `agg_name` instead of `k`, `agg_definition` instead of `v`, otherwise it gets a bit confusing.

Regarding the implementation, I believe it will be more explicit and understandable when we will extend this to support sub-aggregations (https://github.com/cernopendata/opendata.cern.ch/issues/3023).